docs/components/topnav-primary/test.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Testing - Top Navigation
+---
+{% extends 'no-frame.njk' %}
+
+{% block content %}
+<div id="vue-eyebrowDemo">
+  <div id="hxTopnav-primary">
+    <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+    <div id="right-menu">
+      <div>
+        <a href="#" @click="selectedTag(tab)" :current-tab="currentTab">
+          <hx-icon type="bell"></hx-icon>
+          <p>Notifications</p>
+        </a>
+
+        <a href="#" class="selected">
+          <hx-icon type="ticketing"></hx-icon>
+          <p>Tickets</p>
+        </a>
+
+        <a href="#">
+          <hx-icon type="support"></hx-icon>
+          <p>Support</p>
+        </a>
+
+        <a href="#" class="disabled">
+          <hx-icon type="billing"></hx-icon>
+          <p>Billing</p>
+          <hx-tooltip  position="top-center">
+            Top Center Tooltip
+          </hx-tooltip>
+        </a>
+
+        <a href="#">
+          <hx-icon type="account"></hx-icon>
+          <p>Account</p>
+        </a>
+      </div>
+      <div class="straightLine"></div>
+      <div>
+        <hx-disclosure id="topnav-menu" aria-controls="eyebrowMenuId" role="button" aria-expanded="true" class="disabled">
+          <span>Drop-down label</span>
```

Comment:
Please change to `Jane User` as the username, and also add **My Profile and Settings** to the demo/test pages:

<img width="490" alt="Screen Shot 2020-09-28 at 4 49 35 PM" src="https://user-images.githubusercontent.com/10120600/94494388-b0fc5500-01b4-11eb-8b62-da1bdacb2906.png">

Reply:
Changes have been done, please review.